src/lwc/geListView/geListView.js

```diff
@@ -1,7 +1,7 @@
 import { LightningElement, api, track, wire } from 'lwc';
 import { getObjectInfo } from 'lightning/uiObjectInfoApi';
 import { dispatch, handleError, format, generateId, sort, deepClone, showToast } from 'c/utilTemplateBuilder';
-import CumulusStaticResources from 'c/utilCumulusStaticResources';
+import LibsMoment from 'c/libsMoment';
```

Comment:
@ptewson-sfdo - Took a closer look at this and ran into some issues with the resulting references from the loadScript method With that said, I'd actually like to make this component _more_ specific and make it responsible for only importing the moment.js library. I've renamed it to reflect that here and I think makes what this component does more clear. Imo we should either do single responsibility library import components like this or have one that imports all the resources that we ship in cumulus static resources. Maybe we can take a look at creating a more generic one at a later time?

Reply:
I think this all makes sense, and thanks for looking into that - I think given those findings, updating this to be more specific is the right way to go!